app/javascript/dashboard/routes/dashboard/settings/attributes/CustomAttribute.vue

```diff
@@ -123,6 +170,33 @@ export default {
     fetchAttributes(index) {
       this.$store.dispatch('attributes/get', index);
     },
+    async deleteAttributes({ id }) {
+      try {
+        await this.$store.dispatch('attributes/delete', id);
+        this.showAlert(this.$t('ATTRIBUTES_MGMT.DELETE.API.SUCCESS_MESSAGE'));
+      } catch (error) {
```

Comment:
Check any message exist in the `error` object. Otherways show the default error message